apps/fluent-tester/src/TestComponents/Svg/SvgTest.tsx

```diff
@@ -248,6 +248,7 @@ const RemoteSvgTest: React.FunctionComponent = () => {
           style={styles.svg}
           width="100"
           height="100"
+          // eslint-disable-next-line @microsoft/sdl/no-insecure-url
```

Comment:
For adding new suppressions, could you add a comment mentioning why it's ok to bypass?

Reply:
Yes, I just deleted this code. It was disabled already so removed this will address issue #1596 (or make it no longer relevant)